packages/services/src/index.ts

```diff
@@ -159,11 +160,12 @@ const dsmBots = new DevNetServiceConfig({
 
 const csm = new DevNetServiceConfig({
   repository: {
-    url: "git@github.com:lidofinance/community-staking-module.git",
```

Comment:
We probably need to create an abstraction here that allows us to choose how to load repositories. On our server and for some team members, git: is used instead of https:.